File: app/src/main/java/com/sagar/android/logutil/ApplicationClass.java
package com.sagar.android.logutil;

import android.app.Application;

import com.sagar.android.logutilmaster.LogUtil;

public class ApplicationClass extends Application {
    LogUtil logUtil;

    @Override
    public void onCreate() {
        super.onCreate();

        initialiseLogUtil();
    }

    private void initialiseLogUtil() {
        LogUtil.Builder builder = new LogUtil.Builder()
                .setCustomLogTag("Custom_Log_Tag")
                .setShouldHideLogInReleaseMode(true, BuildConfig.DEBUG);
        logUtil = builder.build();
    }

    public LogUtil getLogUtil() {
        return logUtil;
    }
}

File: app/src/main/java/com/sagar/android/logutil/ApplicationClass.kt
package com.sagar.android.logutil

import android.app.Application

import com.sagar.android.logutilmaster.LogUtil

class ApplicationClass : Application() {
    lateinit var logUtil: LogUtil

    override fun onCreate() {
        super.onCreate()

        initialiseLogUtil()
    }

    private fun initialiseLogUtil() {
        val builder = LogUtil.Builder()
            .setCustomLogTag("Custom_Log_Tag")
            .setShouldHideLog(false)
        logUtil = builder.build()
    }
}
